docs/programme/area-owners.md

```diff
@@ -0,0 +1,150 @@
+# Area Owners
+
+In order to make sure that nothing falls through the cracks, and that work is
+not unduly piled onto a single person, we have a system of area owners. Ideally
+these area owners cover all parts of the programme. This list is not set in
+stone, and we expect it to change over time.
+
+It is wise to make sure that each area owner has a deputy, who can step in when
+they are unavailable, and can at least be aware of what is going on in their
+area.
+
+## Health and Safety
+
+Responsibilities:
+
+* Safety and compliance inspections
+* Risk assessments
+* Area wardens
+* Fire safety
+
+## Media / Press
+
+Responsibilities:
+
+* Photos
+* Organisation of group photo
+* Blog posts/announcements
+* Social media (incl. competitor social media)
+* Inviting guests
+* Arranging guest speaker
+* Swag
+* Competition booklet
+* Wristbands/lanyards
+* Reception
+* Media consent arrangements
+* Website (NB srcomp)
+
+## Mentoring
+
+Responsibilities:
+
+* In schools
+* Helpdesk
+* First-line support
+* Roving help at competition
+* Microgames
+* Discord bot & moderation
+* Tech day content/structure
+* Power tools area
+
+## Teams
+
+Responsibilities:
+
+* Supervisor interaction
+* Signups
+* Tracking progress
+* Announcements
+* Challenges
+* Feedback & surveys
+
+## Kit
+
+Responsibilities:
+
+* Kit handout
+* Replacements
+* Documentation
+* Handback/recovery/return
+* Overnight kit loans
+* Second-line support
+* Battery charging
+
+## Event logistics
+
+Responsibilities:
+
+* Setup
+* Teardown
+* Tech hire
+* Venue interaction
+* Kickstart venues
+* Tech day venues
+* Lighting rig
+* In-venue audio
+* Arena props & build
+* Map
+* Make venue secure & safe overnight
+* In-venue signage
+
+## Live streaming
+
+Responsibilities:
+
+* Live stream
+* Live stream tech
+* Live stream commentators
+* Virtual competition broadcast
+
+## Production
+
+Responsibilities:
+
+* Prizegiving ceremony
+* Kickstart talk
+* Competition briefings
+* In-competition announcements
+* Music
+* PRS/PPL
+* Mixtape
+* Prizes
+* Certificates if applicable
+
+## Game
+
+Responsibilities:
+
+* Judging
+* Match operations
+* Score entry
+* srcomp
+* Scheduling (match and day)
+* Rules
+
+## Volunteer coördination
```

Comment:
Please add "coördination" to the `.spelling` file in a suitable section.

Reply:
ecae560b0b4158ed7d10c151035464df2857a97a